portfolio/src/main/webapp/index.html

```diff
@@ -291,5 +299,21 @@ <h3>Bourbon Village in Morocco</h3>
         </div>
       </div>
     </div>
+
+    <!-- Comments section -->
+    <div class="page-content inactive" id="comments">
+      <div id="hello-container">
+        <div id="comments-list"></div>
+        <br/><br/><br/>
+        <form action="/data" method="POST" target="dummyframe">
+          <p>Add your own comments!</p>
+          <input type="text" name="user-comment">
+          <br/><br/>
+          <!-- TODO: fix race condition, times where fetch completes before post -->
```

Comment:
Do you have a strategy in mind how to fix this?

Reply:
One possibility is to route the POST request through javascript instead of having the form to handle it for me. This might be the simplest solution. Do you have anything in mind that might help out?

Reply:
Why don't you give it a try? This is what is called AJAX.

Reply:
We can close this PR first and later you can create another PR for the experiment. Let's target to meet all the PRs.

Reply:
I agree, I'll try it out, thanks!